examples/feature_demo/gltf_viewer.py

```diff
@@ -27,7 +27,11 @@
 
```

Comment:
You are doing so much work on gltf!

is it worth adding some gltf validation examples?

Reply:
I have an open mind on this. Currently, pygfx's support for advanced rendering features in the glTF specification is validated by comparing it against the model asset library and test cases provided by the Khronos Group. Additionally, each major advanced rendering feature has its own dedicated example. The screen validation examples in pygfx primarily serve to prevent unintended modifications to rendering effects during the development of other features. However, in the current development process, most changes to rendering effects are intentional.